### ISSUE TITLE
compatibility with Firefox on FreeBSD

### ISSUE THREAD

**Issue by grahamperrin** [totallymike/contextPlus#8]
Containers are listed in the _Move to context_ submenu but when a context (container) is selected, no move occurs. 

With <kbd>Control</kbd> whilst selecting a context, no copy occurs. 

I tried installations for Linux, for Mac OS X and for Windows. 

## Environment

- Containers Experiment 2.3.0
- Firefox 53.0.2 (64-bit) on TrueOS Desktop, which is based on FreeBSD 12.0-CURRENT
- http://www.freshports.org/www/firefox/

```text
$ pkg info firefox
firefox-53.0.2,1
Name           : firefox
Version        : 53.0.2,1
Installed on   : Sun May 28 15:09:33 2017 BST
Origin         : www/firefox
Architecture   : FreeBSD:12:amd64
Prefix         : /usr/local
Categories     : ipv6 www
Licenses       : 
Maintainer     : gecko@FreeBSD.org
WWW            : http://www.mozilla.com/firefox
Comment        : Web browser based on the browser portion of Mozilla
Options        :
        ALSA           : on
        BUNDLED_CAIRO  : on
        CANBERRA       : off
        DBUS           : on
        DEBUG          : off
        DTRACE         : on
        FFMPEG         : on
        GCONF          : off
        GNOMEUI        : off
        INTEGER_SAMPLES: off
        LIBPROXY       : off
        OPTIMIZED_CFLAGS: on
        PGO            : off
        PROFILE        : off
        PULSEAUDIO     : on
        RUST           : on
        SNDIO          : off
        TEST           : off
Shared Libs required:
        libdbus-glib-1.so.2
        libgtk-x11-2.0.so.0
        libXext.so.6
        libpango-1.0.so.0
        libjpeg.so.8
        libpixman-1.so.0
        libsqlite3.so.0
        libicuuc.so.58
        libcairo.so.2
        libgtk-3.so.0
        libffi.so.6
        libxcb-shm.so.0
        libX11-xcb.so.1
        libfreetype.so.6
        libcairo-gobject.so.2
        libatk-1.0.so.0
        libhunspell-1.6.so.0
        libevent-2.1.so.6
        libvpx.so.4
        libplc4.so
        libXcomposite.so.1
        libxcb.so.1
        libicui18n.so.58
        libgio-2.0.so.0
        libssl3.so
        libXfixes.so.3
        libnss3.so
        libgdk_pixbuf-2.0.so.0
        libogg.so.0
        libv4l2.so.0
        libgobject-2.0.so.0
        libnssutil3.so
        libplds4.so
        libharfbuzz.so.0
        libstartup-notification-1.so.0
        libX11.so.6
        libdbus-1.so.3
        libXdamage.so.1
        libnspr4.so
        libXt.so.6
        libgraphite2.so.3
        libglib-2.0.so.0
        libpng16.so.16
        libXrender.so.1
        libvorbis.so.0
        libgdk-x11-2.0.so.0
        libfontconfig.so.1
        libsmime3.so
        libgdk-3.so.0
Annotations    :
        cpe            : cpe:2.3:a:mozilla:firefox:53.0.2:::::freebsd12:x64
        icon           : http://www.pcbsd.org/appcafe/icons/www_firefox.png
        no_provide_shlib: yes
        screen1        : http://www.pcbsd.org/appcafe/screenshots/www/firefox/screen1.png
        screen2        : http://www.pcbsd.org/appcafe/screenshots/www/firefox/screen2.png
Flat size      : 128MiB
Description    :
Mozilla Firefox is a free and open source web browser descended from the
Mozilla Application Suite. It is small, fast and easy to use, and offers
many advanced features:

 o Popup Blocking
 o Tabbed Browsing
 o Live Bookmarks (ie. RSS)
 o Extensions
 o Themes
 o FastFind
 o Improved Security

WWW: http://www.mozilla.com/firefox
$ 
```

```text
$ about

===================
General information
===================

boot environment now (N) … 12.0-CURRENT-up-20170602_210748 NR 2017-06-02
       after restart (R) … 12.0-CURRENT-up-20170602_210748 NR 2017-06-02
boot loader …………………………………… BSD
            type ……………………… EFI
CPU ………………………………………………………… Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
    number of cores ……………… 4
host ……………………………………………………… momh167-gjp4-hpelitebook850g2-trueos.university.brighton.ac.uk
memory ………………………………………………… 16384 MB available, 6395 MB free
OS git branch ……………………………………………………………………………………… trueos-stable
OS git revision ………………………………………………………………………………… d7b9fac18
OS kernel build time ………………………………………………………… Mon 2017 May 29 20:21:45 UTC
OS kernel identity …………………………………………… (uname -i) GENERIC
OS platform (architecture) ……………………… (uname -m) amd64
OS release level ………………………………………………… (uname -r) 12.0-CURRENT
OS version and patch level …… (freebsd-version) 12.0-CURRENT
TrueOS package set ………………… CUSTOM
TrueOS version …………………………… TrueOS-Desktop-201706021450
uptime ………………………………………………… 1:40
user  …………………………………………………… grahamperrin

More (TrueOS Desktop):

    desktop environment …… KDE
    sound card driver ………… pcm1: <Realtek ALC280 (Analog 2.0+HP/2.0)> on hdaa1  (1p:1v/1r:1v) default
    wireless driver ……………… iwm0
    X11 drivers ………………………… intel_drv.so 

$ 
```

**Comment by totallymike** [comment]
Thanks! I think I narrowed it down to Firefox 53 not having support `info.modifiers` in contextMenu stuff.  I pushed up a new version which, when approved, should fix it.  Unfortunately by removing the ability to use the `ctrl` key.  

After the update gets published, will you let me know if it works correctly for you?

**Comment by grahamperrin** [comment]
Thanks, the update was not automated (probably because the extension is reportedly not available for FreeBSD) so I removed 0.3.0, installed 0.3.2. 

It works as expected ☑️ with my 'minimal everyday' profile. Thanks!

(Side note: with my more heavily extended 'everyday' profile, 0.3.2 does not work. Probably an extension conflict.)
